core/commands/commands_test.go

```diff
@@ -179,6 +179,7 @@ func TestCommands(t *testing.T) {
 		"/name/pubsub/state",
 		"/name/pubsub/subs",
 		"/name/resolve",
+		"/name/validate",
```

Comment:
I think keeping this in`/name` is fine, but would rename it to  `/name/verify-record`  so it is harder to confuse with `resolve` (+ this is the language we use in specs).